ContriHUB/SecureFace
# Improves access logging and face detection tests

## Description



Additional features are added in test functions:
test_access_logging:
1. Check whether the file exists at the correct path
2. Clean up the log file before running the test

test_face_detection:
1.  Check if the image is not entirely black
2.  Check for Detected Faces Being Non-Overlapping

## Related Issue






Fixes #8 

## Type of change




## Motivation and Context



## How Has This Been Tested?






All tests are successful after running unittest.

## Screenshots (if appropriate):

## Checklist:




- [x] I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).


**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:
